recipes/paintera/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "paintera" %}
+{% set version = "0.17.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 71024837bbfe3a531559a7dbccaffc1f9fd08829ff96ca4d97d841abcf479ea9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
+  skip: True  # [py2k]
```

Could it be ``noarch: python`` ?

---

If it is not dependent of a platform, I think that can be ``noarch``
```suggestion
  noarch: python
```

And you can select python on ``host`` and ``run`` sections

---

Yes, it is platform independent, but it requires python `>=3.6` (which is equivalent to `>=3` for conda-forge as far as I can tell because conda-forge only builds `3.6` and `3.7`). I saw that you also suggested changes to specify the Python version in the `host` and `run` sections. I will go ahead and apply those changes.